aws/resource_aws_fsx_file_system.go

```diff
@@ -0,0 +1,448 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/fsx"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsFsxFileSystem() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsFsxFileSystemCreate,
+		Read:   resourceAwsFsxFileSystemRead,
+		Update: resourceAwsFsxFileSystemUpdate,
+		Delete: resourceAwsFsxFileSystemDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"type": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice([]string{"LUSTRE", "WINDOWS"}, false),
+			},
+			"capacity": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IntAtLeast(300),
+			},
+			"kms_key_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"subnet_ids": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"security_group_ids": {
```

Comment:
This attribute is missing acceptance testing.

Reply:
It would probably help if I finished implementing the attribute :sweat_smile:

Reply:
In this case, `security_group_ids` is only ever used during `Create` actions.  It's not even tracked when you describe a filesystem.  Do you guys have an official way of handling these kinds of attributes?  This seems like an oversight on Amazon's part.